library/ssl_tls.c

```diff
@@ -5659,11 +5652,32 @@ uint16_t mbedtls_ssl_get_tls_id_from_ecp_group_id(mbedtls_ecp_group_id grp_id)
 }
 
 #if defined(MBEDTLS_DEBUG_C)
+static const struct {
+    uint16_t tls_id;
+    const char *name;
+} tls_id_curve_name_table[] =
+{
+    { 25, "secp521r1" },
```

Comment:
Minor: this should use the `MBEDTLS_SSL_IANA_TLS_GROUP_xxx` macros, and (pre-existing) so should the original table.